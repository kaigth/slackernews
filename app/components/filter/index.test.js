import React from 'react';
import Filter from './index';
import renderer from 'react-test-renderer';

it('renders the Filter correctly', () => {
  const tree = renderer
    .create(<Filter />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});