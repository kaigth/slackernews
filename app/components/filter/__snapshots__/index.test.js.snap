// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Filter correctly 1`] = `
<div
  className="list-filter"
>
  <div
    className="_name"
  >
    Sort By: 
  </div>
  <button
    className="button _score _link"
    onClick={[Function]}
    onKeyPress={[Function]}
    tabIndex="0"
  >
    Scores
  </button>
  <button
    className="button _user _link"
    onClick={[Function]}
    onKeyPress={[Function]}
    tabIndex="0"
  >
    Users
  </button>
  <button
    className="button _date _link"
    onClick={[Function]}
    onKeyPress={[Function]}
    tabIndex="0"
  >
    Date
  </button>
  <div
    className="_list-select"
  >
    <div
      className="_name"
    >
      Show as: 
    </div>
    <button
      className="button undefined"
      onClick={[Function]}
      onKeyPress={[Function]}
      tabIndex="0"
    >
      List
    </button>
  </div>
</div>
`;
